images/runtime/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.8
```

Comment:
tempted to use `latest` (which is latest stable) rather than edge / version... for lazy appeal, we won't keep these updated, we're not very disciplined. discuss